package.json

```diff
@@ -31,6 +31,10 @@
     "@types/content-disposition": "^0.5.8",
     "esbuild": "^0.19.5",
     "eslint": "^8.53.0",
+    "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jsdoc": "^48.0.2",
+    "eslint-plugin-require-sort": "^1.3.0",
+    "eslint-plugin-sort-keys-fix": "^1.1.2",
```

Comment:
ESLint already has the [`sort-keys`](https://eslint.org/docs/latest/rules/sort-keys) rule. Why not make use of it?

Reply:
> ESLint already has the [`sort-keys`](https://eslint.org/docs/latest/rules/sort-keys) rule. Why not make use of it?

This is a non-autofix rule. 
Since the properties are already sorted, I can remove this dependency and add the `sort-keys` rule

Reply:
> This is a non-autofix rule. Since the keys are already sorted, I can remove this dependency and add the `sort-keys` rule

Let's go ahead, please.

Reply:
> > This is a non-autofix rule. Since the keys are already sorted, I can remove this dependency and add the `sort-keys` rule
> 
> Let's go ahead, please.

Done.